main_test.go

```diff
@@ -161,6 +212,7 @@ func (f tctxConfigFile) newApp() (*cli.App, *bytes.Buffer) {
 }
 
 func (f tctxConfigFile) Run(t *testing.T, tc TestCase) {
+	t.Helper()
```

Comment:
Marking these as `t.Helper` ensures that failed tests print the proper error line (the call site, not the helper function)